requirements/classes/php2560/packages.R

```diff
@@ -0,0 +1,14 @@
+# Use this script to install packages via CRAN, for example: 
+
+# NOTE: Code below shows how you install R packages from CRAN and Bioconductor. For CRAN packages, you can use 
+# the standard install.packages() function; for Bioconductor packages, however, you 
+# must first install BiocManager and then use that for installs. 
+
+install.packages(c("tidyverse",
```

Comment:
Do you need these?

I think you have `tidyverse`  being installed via CRAN and Conda